elytron/src/test/java/org/wildfly/extension/elytron/TlsTestCase.java

```diff
@@ -345,16 +362,24 @@ public void testSslServiceAuth() throws Throwable {
 
     @Test
     public void testSslServiceAuthTLS13() throws Throwable {
-        Assume.assumeTrue("Skipping testSslServiceAuthTLS13, test is not being run on JDK 11.",
-                System.getProperty("java.specification.version").equals("11"));
+        Assume.assumeTrue("Skipping testSslServiceAuthTLS13, test is not being run on JDK 11+.",
+                JdkUtils.getJavaSpecVersion() >= 11);
         testCommunication("ServerSslContextTLS13", "ClientSslContextTLS13", false, "OU=Elytron,O=Elytron,C=CZ,ST=Elytron,CN=localhost",
                 "OU=Elytron,O=Elytron,C=UK,ST=Elytron,CN=Firefly", "TLS_AES_256_GCM_SHA384", true);
     }
 
+    @Test
+    public void testSslServiceAuthTLS13OpenSsl() throws Throwable {
+        Assume.assumeTrue("Skipping testSslServiceAuthTLS13OpenSsl, test is not being run with JDK 11+ and OpenSSL 1.1.1+",
+                JdkUtils.getJavaSpecVersion() >= 11 && isOpenSSL111OrHigher());
+        testCommunication("ServerSslContextTLS13OpenSsl", "ClientSslContextTLS13OpenSsl", false, "OU=Elytron,O=Elytron,C=CZ,ST=Elytron,CN=localhost",
+                "OU=Elytron,O=Elytron,C=UK,ST=Elytron,CN=Firefly", "TLS_AES_256_GCM_SHA384", true);
+    }
+
     @Test
     public void testSslServiceAuthProtocolMismatch() throws Throwable {
-        Assume.assumeTrue("Skipping testSslServiceAuthProtocolMismatch, test is not being run on JDK 11.",
-                System.getProperty("java.specification.version").equals("11"));
+        Assume.assumeTrue("Skipping testSslServiceAuthProtocolMismatch, test is not being run on JDK 11+.",
+                JdkUtils.getJavaSpecVersion() >= 11);
```

Comment:
@fjuma, just noticed - isn't here missing the `&& isOpenSSL111OrHigher()` part here? There is client using TLS13 two lines down...